.env.dist

```diff
@@ -14,5 +14,6 @@ POSTGRES_DATA_DIR=/var/lib/postgresql/data
 
 REDIS_DATA_DIR=/data
 REDIS_HOST=redis.host
+REDIS_PORT=6379
```

Comment:
This was missing: https://github.com/ZolaApp/zola-api/pull/85/files#diff-321318d6aed05edfbf645099d1a622ffR11.